src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/InactiveAccountsManagementTest.java

```diff
@@ -0,0 +1,315 @@
+    private String adminProvenanceId;
+    private String mediaUserId;
+    private String adminUserId;
+    private String idamUserId;
+    private static final String ACCOUNT_URL = "/account";
+    private static final String ADD_USER_B2C_URL = ACCOUNT_URL + "/add/azure";
+    private static final String ADD_PI_USER_URL = ACCOUNT_URL + "/add/pi";
+    private static final String UPDATE_USER_INFO = ACCOUNT_URL + "/provenance";
+    private static final String NOTIFY_INACTIVE_MEDIA_ACCOUNT = ACCOUNT_URL + "/media/inactive/notify";
+    private static final String DELETE_INACTIVE_MEDIA_ACCOUNT = ACCOUNT_URL + "/media/inactive";
+    private static final String NOTIFY_INACTIVE_ADMIN_ACCOUNT = ACCOUNT_URL + "/admin/inactive/notify";
+    private static final String DELETE_INACTIVE_ADMIN_ACCOUNT = ACCOUNT_URL + "/admin/inactive";
+    private static final String NOTIFY_INACTIVE_IDAM_ACCOUNT = ACCOUNT_URL + "/idam/inactive/notify";
+    private static final String DELETE_INACTIVE_IDAM_ACCOUNT = ACCOUNT_URL + "/idam/inactive";
+    private static final String ADD_SYSTEM_ADMIN_B2C_URL = ACCOUNT_URL + "/add/system-admin";
+    private static final String TESTING_SUPPORT_APPLICATION_URL = "/testing-support/application/";
+    private static final String GET_PI_USER_URL = "/account/%s";
+    private static final String SYSTEM_ADMIN_SSO_URL = ACCOUNT_URL + "/system-admin";
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @BeforeAll
+    public void startUp() throws IOException {
+
+        String requestBody = """
+            {
+                "email": "%s",
+                "provenanceUserId": "%s"
+            }
+            """.formatted(TEST_SYSTEM_ADMIN_EMAIL, UUID.randomUUID().toString());
+
+        OBJECT_MAPPER.findAndRegisterModules();
+
+        headers = Map.of(HttpHeaders.AUTHORIZATION, BEARER + accessToken);
+        String userId =  doPostRequest(SYSTEM_ADMIN_SSO_URL, headers, requestBody)
+            .jsonPath().getString("userId");
+        issuerId = Map.of(ISSUER_ID, userId);
+
+        //ADD MEDIA USER
+        createApplication();
+        AzureAccount mediaUserAzureAccount = createAzureAccount(TEST_EMAIL, Roles.VERIFIED);
+        mediaUserProvenanceId = mediaUserAzureAccount.getAzureAccountId();
+        mediaUserId = createUser(TEST_EMAIL, mediaUserAzureAccount.getAzureAccountId(),
+                                   Roles.VERIFIED, UserProvenances.PI_AAD);
+
+        //ADD SYSTEM ADMIN
+        PiUser systemAdminAccount = createSystemAdminAccount(TEST_ADMIN_EMAIL);
+        adminProvenanceId = systemAdminAccount.getProvenanceUserId();
+        adminUserId = systemAdminAccount.getUserId().toString();
+
+        //ADD IDAM USER
+        idamUserId = createUser(TEST_IDAM_EMAIL, IDAM_USER_PROVENANCE_ID,
+                   Roles.VERIFIED, UserProvenances.CFT_IDAM);
+    }
+
+    @AfterAll
+    public void teardown() {
+        doDeleteRequest(TESTING_SUPPORT_APPLICATION_URL + TEST_EMAIL, headers);
```

Comment:
These need to be the account testing support endpoint rather than the application one